backend/static/frontend/org/riceapps/views/playground_view.js

```diff
@@ -180,5 +192,25 @@ PlaygroundView.prototype.handleChildrenChanged_ = function(event) {
   }
 }
 
+/**
+ * Event handler; called when crn button is clicked. Shows a modal view containing all current CRNs in schedule.
+ * @param {goog.events.BrowserEvent} event
+ * @private
+ */
+PlaygroundView.prototype.onClearPlaygroundClick_ = function(event) {
+  if (event != null) {
+    var new_event = new SchedulePlannerEvent(SchedulePlannerEvent.Type.CLEAR_PLAYGROUND_CLICK);
+    this.dispatchEvent(new_event);
+  }
+}
+
+/**
+ * return the courses (children) in the playground
+ * @override
+ * @return {array}
+ */
+PlaygroundView.prototype.getChildren = function() {
```

Comment:
I'm not sure if this is okay, accessing a private property of goog.ui.Component. What you should do to iterate through children is use a for loop and the methods getChildCount() and getChildAt(i).

See http://docs.closure-library.googlecode.com/git/class_goog_ui_Component.html